CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.2.2
+  - Fix high CPU usage on retries in json_batch mode.
```

Comment:
sorry, one last detail, can you add the this pr's reference to the changelog entry? we have guidelines for editing the changelog [here](https://github.com/elastic/logstash/blob/master/CONTRIBUTING.md#logstash-plugin-changelog-guidelines) (these guidelines are somewhat recent we need to make them easier to find).